pkg/synthetictests/disruption.go

```diff
@@ -117,6 +117,39 @@ func testServerAvailability(
 	}
 }
 
+func TestAPIServerIPTablesAccessDisruption(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "[bz-kube-apiserver] kube-apiserver should be accessible by clients using internal load balancer"
+	namespacesToCount := map[string]int{}
+	messages := []string{}
+	for _, event := range events {
+		reason := monitorapi.ReasonFrom(event.Message)
+		if !strings.Contains(reason, "iptables-operation-not-permitted") {
```

Comment:
This seems brittle, as we will presumably grow other concerning log lines in the future.  And there could also be false-positives where we get `iptables-operation-not-permitted` lines in logs while attempting DNS resolution of things that are not the internal Kube-API domain.  Worth having a specific slug in the `reason` for "kube-apiserver should be accessible by clients using internal load balancer"?

Reply:
> Worth having a specific slug in the reason for "kube-apiserver should be accessible by clients using internal load balancer"?

I figure eventually they'll be motivated to invest here and tweak it however they wish.

Reply:
and reasons are already like APIs in our events.

Reply:
`iptables-operation-not-permitted` is not a clear API for `relevant to  internal Kube-API domain`, but :shrug: fair enough that future changes are cheap.